slides/2024-4S-EASST/index.qmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "The twin peaks of the global and the local: worldwide and modest infrastructures for global health"
+author:
+  - name: "Miller Díaz Valderrama"
+    orcid: "0000-0001-5583-0616"
+  - name: "Natalia Niño"
+    orcid: "0000-0001-7887-9439"
+  - name: "Catalina González-Uribe"
+    orcid: "0000-0002-3322-5017"
+date: "2024-07-19"
+categories: [localism, social science, TRACE-LAC]
+---
+
+Presentation (30min including questions) at the Biannual Conference of the Society for Social Studies of Science (4S) and the European Association for the Study of Science and Technology (EASST) in Amsterdam, under the "Issues of scale: the global and the local in health research projects with a worldwide context" category.
+
+## Abstract
+
+Global health projects have long emphasized the digital transformation of Global South countries to align their contexts with global infrastructures. Much of this emphasis is done by pointing at these contexts as flawed: interoperability, standards and classifications, and human capacity as well are seen as insufficient, too local, and disorganized. Consequently, international funding is directed towards enhancing capacity and infrastructure in these regions to facilitate smoother data exchange, with digital transformation promising seamless transitions across different scales. What happens if we conduct an inversion of this narrative? Could we follow global infrastructures in their flaws when meeting "the local"?
+
+Based on a robust tradition of reflections in Science and Technology Studies, Critical Global Health, and Critical Data Studies about context (De Laet M., 2000; Tsing, A., 2005; Adams, 2016; Biehl 2016; Montgomery, C. M., Kingori, P., Sariola, S., & Engel, N, 2017; Lee, F 2021; 2023; Hepp, A; Jarke J; Kramp L 2022), we propose a symmetrical understanding of local-global relations in Global Health projects and enterprises. For this, we offer some examples in which we can see the operation of this kind of understanding, drawing from our involvement with two global projects on emergency preparedness, targeting the production of technologies for the analysis and control of infectious diseases.
```

Comment:
Do we want/need to include the exact references, as we can with a bib file?

Reply:
It would be nice in theory but probably not worth the extra effort